app/services/user_alerts/alert_user_about_personal_key_sign_in.rb

```diff
@@ -1,14 +1,23 @@
 module UserAlerts
   class AlertUserAboutPersonalKeySignIn
+    # @return [FormResponse]
     def self.call(user, disavowal_token)
-      user.confirmed_email_addresses.each do |email_address|
+      emails = user.confirmed_email_addresses.map do |email_address|
         UserMailer.personal_key_sign_in(
           email_address.email, disavowal_token: disavowal_token
         ).deliver_now
       end
-      MfaContext.new(user).phone_configurations.each do |phone_configuration|
+      telephony_responses = MfaContext.new(user).phone_configurations.map do |phone_configuration|
         Telephony.send_personal_key_sign_in_notice(to: phone_configuration.phone)
       end
+      FormResponse.new(
+        success: true,
+        errors: {},
+        extra: {
+          emails: emails.count,
+          sms_message_ids: telephony_responses.map { |resp| resp.to_h[:message_id] },
```

Comment:
In a few places like this, I opted to call this `sms_message_ids` to differentiate from the emails we also send in the same location, so we could add email IDs in the future if we wanted